agentic/scripts/measure-context-budget.py

```
@@ -0,0 +1,240 @@
+    """Count non-empty lines in a file."""
+    try:
+        with open(file_path, 'r', encoding='utf-8') as f:
+            lines = [line.strip() for line in f if line.strip()]
+            if lines and lines[0] == '---':
+                try:
+                    end_idx = lines[1:].index('---') + 2
+                    lines = lines[end_idx:]
+                except ValueError:
+                    pass
+            return len(lines)
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)